### DISCUSSION TITLE
Reflashing SNES Repros: LoROM up to 8MB ?!

### DISCUSSION THREAD

**Discussion by StanMOI** [sanni/cartreader#892] [General Chat]
Hi Sanni,
On "[Reflashing SNES Repros](https://github.com/sanni/cartreader/wiki/Reflashing-SNES-Repros)" 
you noting that "_LoRom (P0) will write up to 8MB starting from file 0x0 to flashrom 0x0."_
8MB ?! Are you sure? 
LoROM = up to 8MB
and 
HiROM = up to 4MB

Can you please also note on this page HOW to wire (step by step), ExLoROM and exHiROM files.
Thanks

**Comment by sanni** [comment]
See this discussion: https://github.com/sanni/cartreader/discussions/164#discussioncomment-845136
